test/integration/host_vars/aws_local.yml

```diff
@@ -0,0 +1,2 @@
+ansible_connection: local
+ansible_python_interpreter: "{{ ansible_playbook_python }}"
```

Comment:
Files specific to a test target should be under that target's directory tree, not directly under `test/integration/`.